apps/vscode-wing/.projenrc.ts

```diff
@@ -153,10 +153,10 @@ const contributes: VSCodeExtensionContributions = {
 };
 
 project.addFields({
-  publisher: "Monada",
+  publisher: "Wing Cloud",
```

Comment:
This might be a problem, changing the publisher name is a process we'd have to look into

Reply:
also I don't think it can have a space in it.

This publisher name becomes part of the id of the extension

Reply:
So people would have to re-download the extension

Reply:
Good catch https://stackoverflow.com/questions/42745643/how-to-change-publisher-name-of-already-published-extension

I'll this change

Reply:
https://github.com/winglang/wing/issues/3422